packages/flutter_tools/test/general.shard/ios/xcodeproj_test.dart

```diff
@@ -23,28 +24,52 @@ import '../../src/mocks.dart' as mocks;
 const String xcodebuild = '/usr/bin/xcodebuild';
 
 void main() {
+  FakeProcessManager fakeProcessManager;
```

Comment:
Just moved the declarations and -setUp above the `MockProcessManager ` group that was declaring the same variables.